src/main/java/seedu/foodrem/logic/commands/itemcommands/EditCommand.java

```diff
@@ -28,25 +27,13 @@
  * Edits the details of an existing item in FoodRem.
  */
 public class EditCommand extends Command {
-
-    public static final String COMMAND_WORD = CommandWord.EDIT_COMMAND.getValue();
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates the details of the item identified "
-            + "by the index number used in the displayed item list. "
-            + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
-            + "[" + CliSyntax.PREFIX_NAME + "NAME] "
-            + "[" + CliSyntax.PREFIX_ITEM_QUANTITY + "QUANTITY] "
-            + "[" + CliSyntax.PREFIX_ITEM_UNIT + "UNIT] "
-            + "[" + CliSyntax.PREFIX_ITEM_BOUGHT_DATE + "BOUGHT DATE] "
-            + "[" + CliSyntax.PREFIX_ITEM_EXPIRY_DATE + "EXPIRY DATE]...\n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + CliSyntax.PREFIX_ITEM_QUANTITY + "1000 "
-            + CliSyntax.PREFIX_ITEM_QUANTITY + "grams";
-
-    public static final String MESSAGE_EDIT_ITEM_SUCCESS = "Edited Item: %1$s";
+    // Workaround checkstyle violation
```

Comment:
May I check what's being violated here?

Reply:
Checkstyle doesn't allow FIXME comments (they will give TODO preferred over FIXME):

![telegram-cloud-photo-size-5-6221972818405470784-y](https://user-images.githubusercontent.com/34370238/196122878-af1f7058-a3a1-46e0-8a3c-e350f8646279.jpg)

Looking into the `checkstyle.xml`, we see the regex for this rule. Adding a space fixes that, but we still need a way to track items to be done, hence added a TODO as well.